model/Building.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+
+const buildingSchema = mongoose.Schema({
+  picture: {
+    type: [String],
+  },
+  auctionNumber: {
+    type: String,
+    required: true,
+  },
+  address: {
+    type: String,
+    required: true,
+  },
+  buildingType: {
+    type: String,
+    enum: ['아파트', '주택', '다세대/다가구', '오피스텔/원룸'],
+    default: '아파트',
+    required: true,
+  },
+  squareMeters: {
+    type: String,
+    required: true,
+  },
+  connoisseur: {
+    type: String,
+    required: true,
+  },
+  lowestPrice: {
+    type: String,
+  },
+  deposit: String,
+  process: {
+    type: [Object],
+    default: null,
```

Comment:
```suggestion
  process: {
    type: [processFieldSchema],
    default: null,
```

서브도큐먼트를 만들어서 사용하는게 좋을 것 같습니다.